examples/offline_inference_vision_language.py

```diff
@@ -87,6 +87,7 @@ def run_phi3v(question, modality):
         model="microsoft/Phi-3-vision-128k-instruct",
         trust_remote_code=True,
         max_num_seqs=5,
+        processor_kwargs={"num_crops": 16},
```

Comment:
Can you link to the HF repo explaining how to use `num_crops`?

Please update the multi-image input example as well.